test/dst/model.go

```diff
@@ -143,6 +143,10 @@ func (m *Model) Step(t int64, req *t_api.Request, res *t_api.Response, err error
 			return nil
 		case t_api.ErrAIOSubmissionQueueFull:
 			return nil
+		case t_api.ErrAIOStoreFailure:
+			if strings.Contains(resErr.Unwrap().Error(), "aio dst: failure") {
```

Comment:
don't do trings. do errors.Is or errors.As if you can. suggestion starting error:

Reply:
.unwrap() no .Error()